filefn/python/filefn/server/processing/service.py

```
@@ -0,0 +1,394 @@
+                            "versionId": input["versionId"],
+                            "storageKey": input["storageKey"],
+                            "mimeType": input["mimeType"],
+                            "size": input["size"],
+                            "fileName": input["fileName"],
+                            "tenantId": input.get("tenantId"),
+                            "idempotencyKey": idempotency_key,
+                        }
+                    )
+                    return {"enqueued": True, "jobId": result.get("jobId")}
+                except Exception:
+                    raise errors.processing_enqueue_failed()
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)